### ISSUE TITLE
how do clippy args work?

### ISSUE THREAD

**Issue by danieleades** [qernal/github-actions-rust-clippy#24]
i'm attempting to add an `--all-targets` flag to this action like so-

```yaml
      - uses: actions/checkout@v3
      - uses: qernal/github-actions-rust-clippy@v2.0.0
        with:
          args: --all-targets
          rust_version: nightly-2022-10-18
```

but i'm getting this error-

```
docker: Error response from daemon: failed to create shim: OCI runtime create failed: runc create failed: unable to start container process: exec: "--all-targets": executable file not found in $PATH: unknown.
```

how should args be passed?

**Comment by andrew-s** [comment]
This should now be fixed, published version `v2.0.1` - multiple arguments are now comma separated.
I don't think a comma as a separator should be a problem, let me know if this fixes your problem

**Comment by danieleades** [comment]
> This should now be fixed, published version `v2.0.1` - multiple arguments are now comma separated.
> I don't think a comma as a separator should be a problem, let me know if this fixes your problem

Thanks for addressing this.
Comma-separated args is inconsistent with the way args are specified in other similar GitHub actions. I would expect these to be space-separated (matching the behaviour of the command line)

**Comment by andrew-s** [comment]
Yeah absolutely, that makes a lot more sense.
I've changed this to now be `shlex.split()` in `v2.0.2` - I've tested with a few parameters, maybe there's some edge cases where this is problematic.

Let me know if this fixes the problem.
